src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_app_controller.tsx

```diff
@@ -148,11 +148,20 @@ export class DashboardAppController {
       history,
     });
 
-    const stopSyncingAppFilters = syncAppFilters(filterManager, {
-      set: filters => dashboardStateManager.setFilters(filters),
-      get: () => dashboardStateManager.appState.filters,
-      state$: dashboardStateManager.appState$.pipe(map(state => state.filters)),
-    });
+    // sync initial app filters from state to filterManager
+    filterManager.setAppFilters(_.cloneDeep(dashboardStateManager.appState.filters));
+    // setup syncing of app filters between appState and filterManager
+    const stopSyncingAppFilters = connectToQueryState(
+      queryService,
+      {
+        set: ({ filters }) => dashboardStateManager.setFilters(filters || []),
```

Comment:
It looks like we can just pass the actual state container used by `dashboardStateManager` down here instead of emulating it like this and it would do the same thing - not required though